frontend/main/src/layout/Footer.tsx

```diff
@@ -111,15 +111,20 @@ export default function Footer({
           <section className="grid grid-cols-1 gap-4 lg:justify-self-end 2xl:justify-self-center">
             <h4 className="underline whitespace-nowrap">Helpful Links</h4>
             <div className="grid w-64 grid-cols-2 gap-4">
-              {site?.pageLinks?.map((pageLinks) => (
-                <a
-                  href={pageLinks.slug}
-                  className="links-secondary"
-                  key={pageLinks.slug}
-                >
-                  {pageLinks.title}
-                </a>
-              ))}
+              {site?.pageLinks?.map((pageLink) =>
```

Comment:
I miss understood where you wanted to add this.
I want to drive this from the backend if we want to do this it should also open in a separate Tab. We might need to adjust that schema so that we can mark a link as "external" then it will handle all of this type of logic.